app/views/medications/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<h3>Medication information</h3>
+<p>Brand Name: <%= @medication.brand_name %></p>
+<p>Product NDC: <%= @medication.product_ndc %></p>
+<% if @medication.symptoms.empty? == false %>
```

Comment:
this could be slightly cleaned up by saying `unless @mediation.symptoms.empty?`

Reply:
Sure thing, good catch.